app/controllers/import/prospects_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Import
+  class ProspectsController < ApplicationController
+    include ActionView::Helpers::TextHelper
+
+    def create
+      prospects = ProspectApiWrapper.instance.get_prospects
+      created_prospects = Prospect.create(prospects.reverse)
+      size = created_prospects.select(&:id).count
+
+      flash[:notice] = "#{pluralize(size, 'new prospect')} #{'was'.pluralize(size)} imported"
```

Comment:
More flexible approach would be to use I18n here http://guides.rubyonrails.org/i18n.html#pluralization